RadixWallet/Clients/AppEventsClient/AppEventsClient+Live.swift

```diff
@@ -0,0 +1,18 @@
+extension AppEventsClient: DependencyKey {
+	public static let liveValue: AppEventsClient = {
+		@Dependency(\.homeCardsClient) var homeCardsClient
+
+		return .init(
+			handleEvent: { event in
+				switch event {
```

Comment:
I liked this idea a lot, so gave it a try [here](https://github.com/radixdlt/babylon-wallet-ios/pull/1187/commits/e1cf6f9e96b267ce9864e2c247e2c7dafad06119). However, there seems to be a race condition error since the `.appStarted` event is never received by the listener. I even added [this](https://github.com/radixdlt/babylon-wallet-ios/pull/1187/commits/e1cf6f9e96b267ce9864e2c247e2c7dafad06119#diff-4c2af50d3b79c7cb4b1f6a5f61abfbbfa369508a1c44ac6a574379002aced77aR29) handy method to be called by the `AppEventsClient` on its initialization, to make sure the `HomeCardsClient` subscribes to events before the first one is dispatched, but problem was still present.

We can work on some delays but even having the `AppsEventsClient` _starting_ the `HomeCardsClient` (and whatever other listener client we would have in the future) goes against the whole reasoning behind this change.

Reply:
resolved with `AsyncReplaySubject`